### ISSUE TITLE
Tracking all order books simultaneously

### ISSUE THREAD

**Issue by msieg** [ccxt/ccxt#932]
Is it possible to read the order book for every coin on binance simultaneously, or do I need to loop through to get each one?

- OS: Macook
- Language: Python
- CCXT version:
- Exchange: Binance
- Method: 

```
book = exchange.fetch_order_book('Symbol') vs. book = exchange.fetch_order_book(<all_coins>)
```

**Comment by kroitor** [comment]
> Is it possible to read the order book for every coin on binance simultaneously, or do I need to loop through to get each one?

It is not possible with REST, you need to loop through to get each one. With WS this may be possible, we are working on unifying WS right now. Let me know if this answers your question or not. Thx!

**Comment by msieg** [comment]
This does answer my question, thanks! Please let us know when WS is ready!

**Comment by kroitor** [comment]
Sure!

**Comment by kroitor** [comment]
Good news! Merged #1050 into ccxt version 1.10.1024!
